### PR TITLE
Joey workbench

### PR DESCRIPTION
**Pull request by jrfeffer**
Hey I've made a profile page basically just by mimicking some of the other pages. It has a search bar that will allow you to search for other users but the functionality of this hasn't been added yet as this is just the prototype.

### CODE REVIEW

**Comment by hmontero1205** [comment]
Looks good to me!
Just one thing -- I think you forgot to add links to your profile page.
Other than that, I guess we'll wait on Tung to merge everything?

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  <div id="container">
+
+    <% if(request.getAttribute("error") != null){ %>
+        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
+    <% } %>
+
+    <% if(request.getSession().getAttribute("user") != null){ %>
+      <h1>Search for a Friend!</h1>
+      <form action="/conversations" method="POST">
```
Comment (pnvt23) [inline src/main/webapp/WEB-INF/view/profile.jsp]: can you add a comment why you want to redirect user to conversations/ 

It looks like we are trying to search for a user though ?

**Comment by jrfeffer** [comment]
that was something that I forgot to change when I copied over some code from the conversations.jsp. I've just changed it to reflect the fact that this is performing actions on the profile pages.

**Review by pnvt23 — APPROVED** [review]
Looks good to me

**Pull request closed without merging**